docs/user-guide/faq.md

```diff
@@ -186,14 +191,18 @@ If you need to reference an old version of a dataset, you can find previous vers
 Data from AACR Project GENIE are provided in a [dedicated instance of cBioPortal](https://www.cbioportal.org/genie/). You can also download GENIE data from the [Synapse Platform](https://synapse.org/genie). Note that you will need to register before accessing the data. Additional information about AACR Project GENIE can be found on the [AACR website](https://www.aacr.org/Research/Research/Pages/aacr-project-genie.aspx).
 
 ### TCGA
-#### How does TCGA data in cBioPortal compare to TCGA data in Genome Data Commons?
-We do not currently load the mutation data from the GDC. Instead, we have the original mutation data generated by the individual TCGA sequencing centers. The source of the data is the Broad Firehose (or the publication pages for data that matches a specific manuscript). These data are usually a combination of two mutation callers, but they differ by center (typically a variant caller like MuTect plus an indel caller), and sequencing centers have modified their mutation calling pipelines over time.
+#### How do the TCGA studies sourced from Genomic Data Commons (GDC) compare to the other TCGA datasets? Which one should I use?
```

Comment:
I think we should should leave the existing section "How does TCGA data in cBioPortal compare to TCGA data in Genome Data Commons?" , but maybe rename it to "How does TCGA Firehose Legacy data in cBioPortal compare to TCGA data in Genome Data Commons?".  Then we create a new section to describe the GDC data source from ISB-CGC.  Lets talk more about this at the standup.

Reply:
Do we have any links to how GD. - mutation is generated?  To ISB or GDC?